expression/builtin_json.go

```diff
@@ -766,7 +769,90 @@ type jsonKeysFunctionClass struct {
 }
 
 func (c *jsonKeysFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "JSON_KEYS")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, errors.Trace(err)
+	}
+	argTps := []types.EvalType{types.ETJson}
+	if len(args) == 2 {
+		argTps = append(argTps, types.ETString)
+	}
+	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETJson, argTps...)
+	var sig builtinFunc
+	switch len(args) {
+	case 1:
+		sig = &builtinJSONKeysSig{bf}
+	case 2:
+		sig = &builtinJSONKeys2ArgsSig{bf}
+	default:
+		// Should never happens.
+		return nil, errors.Errorf("JSON_KEYS invalid arg length, expect 1 or 2 but got: %v", len(args))
+	}
+	sig.setPbCode(tipb.ScalarFuncSig_JsonKeysSig)
```

Comment:
Seems builtinJSONKeysSig and builtinJSONKeys2ArgsSig should be set different PbCode.

Reply:
@Kingwl PTAL this comment.